### ISSUE TITLE
Black Screen on PS4 Pro, Manjaro

### ISSUE THREAD

**Issue by tonebacas** [thestr4ng3r/chiaki#38]
I'm connected to my PS4 Pro, but I'm not getting any video feed; I get audio and can send input, but I get no video (black screen).

It might be because I'm running on an integrated graphics card on a crappy laptop which doesn't even support OpenGL 3.3, so there's that.

Here's the log in case it's useful:

[chiaki_session_2019-09-28_00-20-30-571571.log](https://github.com/thestr4ng3r/chiaki/files/3664829/chiaki_session_2019-09-28_00-20-30-571571.log)

If you need any further information that could help solve this problem, don't hesitate to ask.

**Comment by thestr4ng3r** [comment]
Yes OpenGL support is the problem here, see the line `OpenGL initialized with version "2.1 Mesa 19.1.5"`. Currently 3.2 Core is required, but I plan to add more options for different kinds of video output.

**Comment by tonebacas** [comment]
Thanks. Yeah, this poor laptop is already showning its age, so these OpenGL issues keep popping up.

Can't wait for the new video output options coming down the road :)

**Comment by hipolito** [comment]
@tonebacas did you tried to use:

`LIBGL_ALWAYS_SOFTWARE=1 chiaki`

Worked just fine, and I use this command for any game that runs on Windows but have some limitation on linux

**Comment by Kimbo4** [comment]
> @tonebacas did you tried to use:
> 
> `LIBGL_ALWAYS_SOFTWARE=1 chiaki`
> 
> Worked just fine, and I use this command for any game that runs on Windows but have some limitation on linux

Hi, where I should set this? 
I have the same issue, and I use manjaro too

**Comment by hipolito** [comment]
Hi @Kimbo4,

I created a script to chiaki using only this:
`
#!/bin/bash
LIBGL_ALWAYS_SOFTWARE=1 chiaki
`
